python/cudf/cudf/_libxx/strings/slice.pxd

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from cudf._libxx.lib import *
+from cudf._libxx.lib cimport *
+from cudf._libxx.aggregation cimport *
+
+cdef extern from "cudf/strings/substring.hpp" namespace "cudf::strings" nogil:
```

Comment:
Lets mirror the files from libcudf and have our `.pxd` files for external declarations follow the `.hpp` files